internal/agent/install_test.go

```diff
@@ -13,56 +11,16 @@ import (
 	"github.com/kairos-io/kairos-agent/v2/pkg/config"
 	fsutils "github.com/kairos-io/kairos-agent/v2/pkg/utils/fs"
 	v1mock "github.com/kairos-io/kairos-agent/v2/tests/mocks"
-	"github.com/twpayne/go-vfs/v4/vfst"
-	"gopkg.in/yaml.v3"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/twpayne/go-vfs/v4/vfst"
 )
 
 const printOutput = `BYT;
 /dev/loop0:50593792s:loopback:512:512:gpt:Loopback device:;`
 const partTmpl = `
 %d:%ss:%ss:2048s:ext4::type=83;`
 
-var _ = Describe("prepareConfiguration", func() {
```

Comment:
I think we should keep a test, even if this one somehow missed the original requirement, at least it makes it clear what we actually want to have working here

Reply:
Got it, yeah. Not sure why I removed the function. I should just adapt it to return an io.reader